ui/app/scripts/deploy.js

```diff
@@ -22,14 +24,23 @@ const //
   PATH_BUILD = `${APP_DIR}/build`,
   TEMPLATES = `../../templates`,
   DEST = `../../static-assets/next`,
-  PLACEHOLDER = '<script id="_placeholderscript_"></script>',
-  indexContents = fs.readFileSync(`${PATH_BUILD}/index.html`).toString(),
-  position = indexContents.indexOf(PLACEHOLDER),
-  templateScripts = indexContents
-    .substr(position + PLACEHOLDER.length)
-    .replace(/\<\/(body|html)>/gi, '')
-    .replace(/<\/script>/gi, '</script>\n'),
-  jsNextScriptsFileContent = '<#include "/templates/web/common/js-global-context.ftl" />\n' + templateScripts;
+  indexContents = fs.readFileSync(`${PATH_BUILD}/index.html`).toString();
+
+let jsNextScriptsFileContent = '<#include "/templates/web/common/js-global-context.ftl" />\n';
+
+const parser = new htmlparser.Parser({
+  onopentag(name, attributes) {
+    if (name === 'script') {
+      jsNextScriptsFileContent += `<script src="${attributes.src}"></script>\n`;
+    } else if (name === 'link' && !attributes.rel.includes('icon')) {
```

Comment:
We should be looking for all the rel=stylesheet. Why are you looking instead at whether it includes icon?

Reply:
im excluding the icon one